cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CertUtils.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import io.strimzi.operator.common.Util;
+
+import java.math.BigInteger;
+import java.security.cert.CertificateEncodingException;
+
+/**
+ * Certificate utility methods
+ */
+public class CertUtils {
+    /**
+     * Generates a short SHA1-hash (a hash stub) of the certificate which is used to track when the certificate changes and rolling update needs to be triggered.
+     *
+     * @param certSecret    Secrets with the certificate
+     * @param key           Key under which the certificate is stored in the Secret
+     * @return              Hash stub of the certificate
+     */
+    public static String getCertificateShortThumbprint(Secret certSecret, String key) {
+        var thumbprint = getCertificateThumbprint(certSecret, key);
+        return thumbprint == null ? null : thumbprint.substring(0, 8);
```

Comment:
out of curiosity, what's the rationale behind `substring(0,8)` and not bigger or shorter?

Reply:
We use the 8 character hash stub for ages. But for whatever reason this Pr has to do it differently and use the full has for the server certificate 🙄

Reply:
Do we have a constant define somewhere for the 8 character size hash stub?

Reply:
I don't think so. Before this PR it was actually just in one place anyway I think.

Reply:
So with two places now, a constant would be better.

Reply:
Done